app/src/main/java/com/example/concordia_campus_guide/Models/RoomModel.java

```diff
@@ -16,9 +16,7 @@
                         childColumns = {"floor_code"},
                         onDelete = CASCADE
                 ),
-                },
-        indices = {@Index(value = {"room_code"},
-        unique = true)})
+                })
```

Comment:
Why did you remove the room_code as being unique ?? Were you having some issues ?

Reply:
You were with me when we did this :P. Yes because "elevators", "womens" and "mens" are not unique.

Reply:
this will cause issues when we link walking points, we should discuss this

Reply:
I agree with thomas cuz I noticed that in the json file we are storing elevators (as the roomCode). Which might be confusing when finding the paths using walkingpoints